websites/S/SeriManga/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "SeriManga",
+    "author": {
+      "name": "Harold",
+      "id": "141206144264699904"
+    },
+    "category": "anime",
+    "logo": "https://i.imgur.com/vWBdu2A.png",
+    "thumbnail": "https://i.imgur.com/BYGkJwg.png",
+    "version": "1.0.0",
+    "color": "#2d2f33",
+    "tags": [
+      "manga",
+      "manhwa"
+    ],
+    "url": "serimanga.com",
+    "description": {
+      "en": "Türkçe Manga ve Çizgi Romanları ücretsiz ve kaliteli bir şekilde okuyabileceğiniz site."
+    },
+    "settings": [
+      {
+        "id": "ID",
+        "multiLanguage": true
+      }
+    ]
```

Comment:
you don't even use it so why you need it?

Reply:
It was there for testing, guess i forgot about it